velox/dwio/common/CachedBufferedInput.cpp

```diff
@@ -452,8 +452,13 @@ class SsdLoad : public DwioCoalescedLoadBase {
       return pins;
     }
     assert(!ssdPins.empty()); // for lint.
-    const auto stats = ssdPins[0].file()->load(ssdPins, pins);
-    updateStats(stats, prefetch, true);
+    try {
```

Comment:
I just realized that CoalescedLoad::loadOrFuture has already handled the async exception so we are good.